tools/webdriver/webdriver/bidi/modules/browser.py

```diff
@@ -27,8 +27,14 @@ def _get_client_windows(self, result: Mapping[str, Any]) -> Any:
         return result["clientWindows"]
 
     @command
-    def create_user_context(self) -> Mapping[str, Any]:
-        return {}
+    def create_user_context(self,
+            accept_insecure_certificate: Optional[bool] = None) -> Mapping[
```

Comment:
nit: maybe we should name all params after the spec name, i.e., `accept_insecure_certs`

Reply:
good point, done. Thanks!